scripts/cipublish.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -ex
+
+vagrant ssh -c "cd /vagrant
```

Comment:
In that case, [Cicero usage](https://github.com/azavea/cicero/blob/develop/scripts/cibuild) needs to be updated as well.